data/economy/taxingareas/index.html

```diff
@@ -68,12 +70,12 @@ <h3 id="TaxAreas">Tax Areas</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
     <img class="inline-text-left outline" src="{% link images/TaxEntities.png %}" alt="Tax Entities" loading="lazy" />
-    <p>Tax Areas 2020 contains the set of taxing areas formed by each unique 2-dimensional intersection of incorporated
+    <p>Tax Areas 2021 contains the set of taxing areas for the year, formed by each unique 2-dimensional intersection of incorporated
       taxing entities in Utah. Each taxing area represents at least a county and school district as members of these two
       categories cover the entire state. The property tax levy rate within a tax area should be the same for a specified
-      property type within that area.</p>
+      property type within that area. 2022 rates have not been done yet.  That won't be till September.</p>
```

Comment:
`till` is not a word.
```suggestion
      property type within that area. 2022 rates have not been done yet.  That data will not be available until September.</p>
```